### PR TITLE
Adds website "microsite" at aerospike.github.io/skyhook

### PR DESCRIPTION
**Pull request by arrowplum**
See https://arrowplum.github.io/skyhook

### CODE REVIEW

**Comment by arrowplum** [comment]
Updated to docs only (got rid of front graphics page until we have something that might look nicer).

 Added more to readme that is included so still just one version between repo readme and pages readme. The blog post is just a copy of the readme markdown as a placeholder. It should be the blog @reugn  wrote for dev.to but I don't have that source.

File: website/docs/0. intro.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+---
+slug: /
+---
```
Comment (reugn) [inline website/docs/0. intro.md:3]: This block is visible from the main readme.

Comment (arrowplum) [reply website/docs/0. intro.md]: ach OK had to do some shuffling to keep the slug out (pure markdown page without frontmatter included by frontmatter version.)

File: website/docs/_intro.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+format. An example configuration file can be found in the [`config`](https://github.com/aerospike/skyhook/blob/a0199da72222984c8417ccaa6e4a02064ed7224b/config/server.yml) folder of this repository.
+If no configuration file is specified, the default settings will be applied.
+
+```text
+[main] INFO  c.a.skyhook.SkyhookServer$Companion - Starting the Server...
+```
+
+Now the server is listening to the `config.redisPort` (default: 6379) and is ready to serve.
+
+If you wish to deploy Skyhook as a cluster of nodes, you can find some example configurations [here](scaling-out).
```
Comment (reugn) [inline website/docs/_intro.md:61]: Broken link here.

File: website/blog/2021-08-30-introducing-skyhook.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+slug: introducing-skyhook
+title: Introducing Skyhook
+tags: [skyhook, redis, aerospike]
+---
+
+# Skyhook
```
Comment (arrowplum) [inline website/blog/2021-08-30-introducing-skyhook.md:7]: @reugn would you replace the contents of this with the markdown contents of your dev.to article?

File: website/docs/_intro.md (hunk 2 above)
Comment (arrowplum) [reply website/docs/_intro.md]: thanks looks like from readme to website absolute link is needed. This is fixed.

File: website/blog/2021-08-30-introducing-skyhook.md (hunk 3 above)
Comment (reugn) [reply website/blog/2021-08-30-introducing-skyhook.md:7]: @arrowplum I will.

**Review by reugn — APPROVED** [review]

**Pull request merged**